config/config.go

```diff
@@ -911,6 +912,7 @@ func NewConfig() (c *Config) {
 		KialiInternal: KialiInternalConfig{
 			CacheExpiration: CacheExpirationConfig{
 				AmbientCheck: 10 * time.Minute,
+				IstioStatus:  30 * time.Minute,
 				Gateway:      4 * time.Minute,
```

Comment:
```suggestion
				Gateway:      4 * time.Minute,
				IstioStatus:  30 * time.Second,
```
I think the unit is wrong here, should be seconds, not minutes, right?

Reply:
Right, good catch!